ui/src/components/settings/sub_pages/audio.rs

```diff
@@ -27,9 +135,11 @@ pub fn AudioSettings(cx: Scope) -> Element {
                 section_description: get_local_text("settings-audio.input-device-description"),
                 no_border: true,
                 Select {
-                    initial_value: "Default".into(),
-                    options: vec!["Default".into()],
-                    onselect: move |_| {}
+                    initial_value: state.read().settings.input_device.as_ref().cloned().unwrap_or("Default".into()),
```

Comment:
`Default` should probably be `default`. I hope it doesn't vary by platform. I have some changes planned for Blink to use a separate code path when the device name is "default"/"Default"